tbconnect/models.py

```diff
@@ -107,11 +120,15 @@ class TBCheck(ExportModelOperationsMixin("tb-check"), models.Model):
     commit_get_tested = models.CharField(
         max_length=3, choices=COMMIT_CHOICES, null=True, blank=True
     )
-    research_consent = models.BooleanField(null=True)
+    research_consent = models.BooleanField(null=True, blank=True)
```

Comment:
why blank=true?

Reply:
It's was part of reverted migrations.

Reply:
I'm more interested to know why we allow it to be anything other than True/False/null ?

Reply:
For miners they won't get the research consent question so their feedback will be None